libvast/src/value_index.cpp

```diff
@@ -482,13 +464,18 @@ sequence_index::sequence_index(vast::type t, size_t max_size)
   };
   value_type_ = caf::visit(f, value_index::type());
   VAST_ASSERT(!caf::holds_alternative<none_type>(value_type_));
+  size_t components = std::log10(max_size_);
+  if (max_size_ % 10 != 0)
+    ++components;
+  size_ = size_bitmap_index{base::uniform(10, components)};
 }
 
 caf::error sequence_index::serialize(caf::serializer& sink) const {
-  return caf::error::eval(
-    [&] { return value_index::serialize(sink); },
-    [&] { return sink(elements_, size_, max_size_, value_type_); }
-  );
+  return caf::error::eval([&] { return value_index::serialize(sink); },
+                          [&] {
+                            return sink(elements_, size_, max_size_,
+                                        value_type_);
+                          });
```

Comment:
`clang-format` needs to be slapped on the fingers here.

Reply:
Can you go into detail? This looks good to me and seems to follow our style guide.

Reply:
In comparison to what we had before, this just looks torn apart.

On a separate note, I don't know why this part of the code got even touched.

Reply:
> On a separate note, I don't know why this part of the code got even touched.

No idea, but I might have touched it by accident.

May I suggest adjusting `.clang-format` like this:
```
PenaltyBreakAssignment: 10
PenaltyBreakBeforeFirstCallParameter: 30
PenaltyBreakString: 40
PenaltyExcessCharacter: 100
PenaltyReturnTypeOnItsOwnLine: 5
```
The function here would be formated like this:
```
caf::error sequence_index::serialize(caf::serializer& sink) const {
  return caf::error::eval(
    [&] { return value_index::serialize(sink); },
    [&] { return sink(elements_, size_, max_size_, value_type_); });
}
```
I don't know if the closing parentheses can be moved back to its own line with another option, but I think I actually like it better this way anyway.

Reply:
Also note that this change to the formatting options would not affect any other code in this PR.
And looking at behaviour returning functions it is actually quite an improvement because the lambdas don't get aligned after the opening curly any more.